lib/mix_test_watch/run.ex

```diff
@@ -0,0 +1,51 @@
+defmodule MixTestWatch.Run do
+  @moduledoc """
+  Runs tests according to given path
+  """
+
+  alias MixTestWatch.Command
+  alias MixTestWatch.Files
+  alias MixTestWatch.Shell
+
+  @spec run(String.t, String.t) :: :ok
+
+  def run(path, config) do
+    case run_single( path, config ) do
```

Comment:
Can we have all the calls to the test running functions here rather than having run_single's failure result in all the tests being run.

i.e.

```elixir
path
|> find_test
|> case do
  :none       -> run_all()
  {:ok, path} -> run_one(path)
end